roles/rciam-metrics/defaults/main.yml

```diff
@@ -102,6 +102,9 @@ metrics_config:
       symlink:
         target: "{{ metrics_path }}/{{ metrics_react_folder_name }}/<tenant_environment>/"
         link: "/var/www/html/<tenant_environment>"
+      # user_id_excludelist:
+      #   user_ids: ""
+
```

Comment:
I'd suggest the following change for adding documentation:
```
# Add user identifiers to this list to exclude specific users from
# metrics related to authentication, user registration, and group
# memberships. These excluded users will not be considered for
# these metrics.
#user_id_excludelist: []
#  - "user1@rciam.example.org"
#  - "user2@rciam.example.org"

```

I haven't included the `user_ids` under `user_id_excludelist`. Is it required to specify the list of user ids?